version 5

```diff
@@ -141,58 +141,12 @@
 							},
 							"hideInAnswer": true
 						}
-					],
-					"condition": {
-						"and": []
-					}
-				},
-				{
-					"render": {
-						"en": "<b>Opening Hours</b>{opening_hours_table(opening_hours)}"
-					},
-					"question": {
-						"en": "Is this the opening hours of {name}?"
-					},
-					"freeform": {
-						"key": "opening_hours",
-						"type": "opening_hours"
-					}
-				},
-				{
-					"render": {
-						"en": "<b><a href=\"tel:{contact:phone}\">{contact:phone}</a></b>"
-					},
-					"question": {
-						"en": "What is this place's phone number?"
-					},
-					"freeform": {
-						"key": "contact:phone"
-					}
-				},
-				{
-					"render": {
-						"en": "<b><a href = \"mailto: {contact:email}\">{contact:email}</a></b>"
-					},
-					"question": {
-						"en": "What is this place's email address?"
-					},
-					"freeform": {
-						"type": "email",
-						"key": "contact:email"
-					}
-				},
-				{
-					"render": {
-						"en": "<b><a href=\"{contact:website}\">Visit website</a></b>"
-					},
-					"question": {
-						"en": "What is this place's website?"
-					},
-					"freeform": {
-						"key": "contact:website",
-						"type": "url"
-					}
+					]
 				},
+				"opening_hours",
```

Comment:
Here, I used the already builtin tagRenderings for contact information. These are already translated and can handle both `email` and `contact:email`, `phone` and `contact:phone`, ...